deploy/manager/10_role_binding.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
```

Comment:
RoleBinding

Reply:
This cannot be RoleBinding because global manager controller needs access to services and secrets in other namespaces. 
When ClusterRole is bound using RoleBinding, it becomes effective only in particular namespace.